housekeeper/store/models.py

```diff
@@ -94,6 +94,9 @@ class File(Model):
     tags = orm.relationship(
         "Tag", secondary=file_tag_link, backref=backref("files", cascade_backrefs=False)
     )
+    archive = orm.relationship(
+        "Archive", back_populates="file", cascade="save-update, merge, delete", uselist=False
```

Comment:
I think the `uselist=False` can be skipped, according to sqlalchemy docs this is usually determined automatically https://docs.sqlalchemy.org/en/20/orm/relationship_api.html#sqlalchemy.orm.relationship.params.uselist

Reply:
I was a bit confused as well, since I did not put it there in the first place but seems as if `file.archive` rerturns an empty list `[]` if not set to False.